<!-- Sensor -->
<element name="sensor" required="0">
  <attribute name="name" type="string" default="__default__" required="1"/>
  <attribute name="type" type="string" default="__default__" required="1"/>
  <attribute name="always_on" type="bool" default="false" required="0"/>
  <attribute name="update_rate" type="double" default="0" required="0"/>
  <attribute name="visualize" type="bool" default="false" required="0"/>

  <element name="origin" required="0">
    <attribute name="pose" type="pose" default="0 0 0 0 0 0" required="1"/>
  </element> <!-- End Origin -->

  <element name="topic" type="string" default="__default" required="0"/>

  <include filename="plugin.sdf" required="*"/>
  <include filename="camera.sdf" required="0"/>
  <include filename="ray.sdf" required="0"/>
  <include filename="contact.sdf" required="0"/>
  <include filename="rfid.sdf" required="0"/>
  <include filename="rfidtag.sdf" required="0"/>

</element> <!-- End Sensor -->
